### DISCUSSION TITLE
What controls the border colour of the Select component?

### DISCUSSION THREAD

**Discussion by mitya33** [tusen-ai/naive-ui#4909] [Q&A] [answered]
In the theme designer, the Select component doesn't have a variable to control its border colour. What does it read from, i.e. what should I set to affect it?

Thanks in advance.

**Comment by jahnli** [comment, marked as answer]
https://www.naiveui.com/zh-CN/os-theme/docs/customize-theme

**Reply by mitya33** [reply]
Could you be more specific? That's a link talking about customising the theme in general.
